plugins/search-backend-module-pg/src/database/types.ts

```diff
@@ -22,6 +22,7 @@ export interface PgSearchQuery {
   fields?: Record<string, string | string[]>;
   types?: string[];
   pgTerm?: string;
+  fallbackSearchQuery?: string;
```

Comment:
Wasn't sure were to put it. Could also expand the PgSearchHighlightOptions instead